examples/vega-example.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>MapD</title>
+    <meta charset="UTF-8">
+    <style>
+      .title {
+        font-weight: bold;
+        text-align:center;
+      }
+      .mapd {
+        position: relative;
+        top: 2px;
+      }
+      .search{
+        displa"y": inline-block;
+        margin-top: 12px;
+        margin-left: 50px;
+      }
+      .data-count {
+        padding-right:20px;
+      }
+      .filter-count {
+        font-weight: bold;
+        color:  #45B1E8;
+      }
+    </style>
+  </head>
+  <body>
+
+    <script src="../thrift/browser/thrift.js"></script>
+    <script src="../thrift/node/mapd.thrift.js"></script>
+    <script src="../thrift/node/mapd_types.js"></script>
```

Comment:
Don't need the `thrift/` files since they're bundled in `dist/browser-connector`. (If you did need them, use the `build/thrift/` versions.)